data/remnant/remnant jobs.txt

```diff
@@ -254,14 +254,15 @@ mission "Remnant: Logistics Civil Engineering"
 		dialog phrase "generic cargo on visit"
 	on complete
 		dialog `A project engineer inspects the <cargo> and signs off on your payment of <payment>.`
-		payment 10000 15
+		payment
+		payment 10000
```

Comment:
OK, so if I'm doing the math right, for a 50 cargo (just a random value between 30 and 80) and a single hyperjump distance, this would give 11500; and now should give 25000 (15000 from the calculation, and +10000 from the flat payment.

Reply:
Since you can't really stack these Remnant jobs much, the more important metric is probably average payment per job, not average payment per ton (which you're effectively comparing when using the same cargo size for both multipliers). Looking at average payment per job, we get the following:

Job | Old | New
---|---|---
Logistics Civil Engineering | 11500 | 20800
Logistics Industrial Delivery | 28250 | 53800
Logistics Outfitter Delivery | 35125 | 58800
Logistics Salvage Delivery | 52625 | 23800

Seeing as how the salvage delivery is the rarest job, it probably shouldn't have the second lowest average pay. The big drop is a result of so much of the current average being a result of the minimum payment of 50k vs the what you gain for the actual transport being only 2.6k on average.

Making changes that bring the average pay to 62600 (without so much of that being front-loaded by the minimum payment).